topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,152 @@
+>
+{: .agenda}
+
+# Preparing Your Data
+
+First, we need to upload the data we'll work with. Ensure that you have the necessary files prepared from Zenodo or a shared data library within Galaxy.
+
+## Data Upload
+
+> <hands-on-title>Data Upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial in Galaxy.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the mitoflash imaging data from [Zenodo](https://zenodo.org/records/14071552) or from the shared data library:
+>    - **Important:** Choose the correct data type if prompted.
+>
+>    ```
+>    https://zenodo.org/records/14071552/files/mitoflashes_8bit.tiff
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets to keep track of them easily, e.g., "MitoFlash."
+> 4. Confirm that the datatypes are correct for each file:
+>    - `mitoflashes_8bit.tif` should be an image tif or tiff file format.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+> 5. Tag each dataset with a label such as "mitoflash" for easy identification later if you are working with multiple images.
+>
+{: .hands_on}
+
+# Analyzing Mitochondrial Movement
+
+In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.
+
+## Step 1: Detecting Mitochondrial Regions
+
+> <hands-on-title>Detecting Mitochondrial Regions</hands-on-title>
+>
+> 1. {% tool [Spot Detection](toolshed.g2.bx.psu.edu/repos/imgteam/spot_detection_2d/ip_spot_detection_2d/0.0.1) %} with the following recommended parameters:
+>    - {% icon param-file %} *"Image input"*: `Mitochondrial_Flashes.tiff`
+>    - **Starting time point** (`frame_1st`): `1` (first frame in the stack)
+>    - **Ending time point** (`frame_end`): `0` (use `0` to process until the last frame)
+>    - **Intensity measurement method** (`typ_intens`): Set to `Smoothed` for a more averaged intensity or `Robust` for resilience to outliers.
+>    - **Threshold** (`thres`): A starting value of `10%` of the global maximum intensity is often effective, but increasing to `15-20%` is possible for higher sensitivity.
+>    - **Gaussian filter sigma** (`ssig`): Use a value around `1` to suppress noise; increase to `1.5` if noise is high.
+>    - **Boundary pixels** (`bndy`): Set to `10` to ignore spots within the image edges, adjusting as needed for image size.
+>    This tool detects mitochondria as spots or 'mitoflashes' based on intensity maxima, identifying them as distinct regions. Adjusting the threshold parameter can improve the accuracy of spot detection, allowing for better identification of mitoflash events.
+>
+{: .hands_on}
+
+## Step 2: Tracking Mitochondrial Movement
```

Comment:
I think it would be good to give a brief motivation here for why tracking of the mitochondria is required in order to analyze their intensity changes over time.